src/algo/max-range/max-usable-range.h

```diff
@@ -0,0 +1,30 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include <types.h>
+
+
+namespace librealsense {
+namespace algo {
+namespace max_range {
+
+class max_usable_range
+{
+public:
+    max_usable_range() = default;
+    // Calculate the maximum usable range based on current ambient light conditions
+    // Input: noise estimation value
+    // Output: maximum usable range [m] 
+    float get_max_range( float nest ) const;
+
+private:
+
+    // Algo parameters
+    const float _processing_gain = 1.75f;
```

Comment:
I like constants as all caps. No need for `_`

Reply:
And perhaps move to .cpp

Reply:
👍